"npm test" fails

emulsify version (see [releases](https://github.com/emulsify-ds/emulsify-drupal/releases)): 4.1.1

node version: v14.17.1

npm (or yarn) version: 6.14.17

**What you did:**

1. `git clone https://github.com/emulsify-ds/emulsify-drupal.git`
2. `cd emulsify-drupal`
3. `git checkout 4.1.1`
4. `npm install`
5. `npm test`

**What happened:**

```
> jest --coverage

Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
FAIL .storybook/setupTwig.test.js
  ● setupTwig › exports emulsifys namespaces

    expect(received).toEqual(expected) // deep equality

    - Expected  - 4
    + Received  + 5

      Object {
    -   "atoms": "../components/01-atoms",
    -   "molecules": "../components/02-molecules",
    -   "organisms": "../components/03-organisms",
    -   "templates": "../components/04-templates",
    +   "atoms": ".././components/01-atoms",
    +   "base": ".././components/00-base",
    +   "molecules": ".././components/02-molecules",
    +   "organisms": ".././components/03-organisms",
    +   "pages": ".././components/04-pages",
      }

      26 |       atoms: '../components/01-atoms',
      27 |       molecules: '../components/02-molecules',
    > 28 |       organisms: '../components/03-organisms',
         |                  ^
      29 |       templates: '../components/04-templates',
      30 |     });
      31 |   });

      at Object.<anonymous> (.storybook/setupTwig.test.js:28:18)

PASS util/loadYaml.test.js
FAIL scripts/a11y.test.js
  ● Test suite failed to run

    ReferenceError: regeneratorRuntime is not defined

      54 | };
      55 |
    > 56 | const lintComponent = async (name) =>
         |                       ^
      57 |   pa11y(`${STORYBOOK_IFRAME}?id=${name}`, {
      58 |     includeNotices: true,
      59 |     includeWarnings: true,

      at scripts/a11y.js:56:23
      at Object.<anonymous> (scripts/a11y.js:66:10)
      at Object.<anonymous> (scripts/a11y.test.js:10:3)

---------------|---------|----------|---------|---------|-------------------
File           | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
---------------|---------|----------|---------|---------|-------------------
All files      |     100 |      100 |     100 |     100 |                   
 .storybook    |     100 |      100 |     100 |     100 |                   
  setupTwig.js |     100 |      100 |     100 |     100 |                   
 util          |     100 |      100 |     100 |     100 |                   
  loadYaml.js  |     100 |      100 |     100 |     100 |                   
---------------|---------|----------|---------|---------|-------------------
Test Suites: 2 failed, 1 passed, 3 total
Tests:       1 failed, 2 passed, 3 total
Snapshots:   0 total
Time:        1.148 s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
```

**Reproduction repository (if necessary):**

**Problem description:**

**Suggested solution:**

--- a/.storybook/setupTwig.js
+++ b/.storybook/setupTwig.js
@@ -31,8 +31,8 @@ const fetchVariantConfig = () => {
         directory: 'components/03-organisms',
       },
       {
-        name: 'pages',
-        directory: 'components/04-pages',
+        name: 'templates',
+        directory: 'components/04-templates',
       },
     ];
   }

--- a/.storybook/setupTwig.js
+++ b/.storybook/setupTwig.js
@@ -16,23 +16,23 @@ const fetchVariantConfig = () => {
     return [
       {
         name: 'base',
-        directory: './components/00-base',
+        directory: 'components/00-base',
       },
       {
         name: 'atoms',
-        directory: './components/01-atoms',
+        directory: 'components/01-atoms',
       },
       {
         name: 'molecules',
-        directory: './components/02-molecules',
+        directory: 'components/02-molecules',
       },
       {
         name: 'organisms',
-        directory: './components/03-organisms',
+        directory: 'components/03-organisms',
       },
       {
         name: 'pages',
-        directory: './components/04-pages',
+        directory: 'components/04-pages',
       },
     ];
   }

--- a/.storybook/setupTwig.test.js
+++ b/.storybook/setupTwig.test.js
@@ -23,10 +23,11 @@ describe('setupTwig', () => {
 
   it('exports emulsifys namespaces', () => {
     expect(namespaces).toEqual({
-      atoms: '../components/01-atoms',
-      molecules: '../components/02-molecules',
-      organisms: '../components/03-organisms',
-      templates: '../components/04-templates',
+      atoms: '.././components/01-atoms',
+      base: '.././components/00-base',
+      molecules: '.././components/02-molecules',
+      organisms: '.././components/03-organisms',
+      pages: '.././components/04-pages',
     });
   });
 });

--- a/.storybook/setupTwig.test.js
+++ b/.storybook/setupTwig.test.js
@@ -23,11 +23,11 @@ describe('setupTwig', () => {
 
   it('exports emulsifys namespaces', () => {
     expect(namespaces).toEqual({
-      atoms: '.././components/01-atoms',
-      base: '.././components/00-base',
-      molecules: '.././components/02-molecules',
-      organisms: '.././components/03-organisms',
-      pages: '.././components/04-pages',
+      atoms: '../components/01-atoms',
+      base: '../components/00-base',
+      molecules: '../components/02-molecules',
+      organisms: '../components/03-organisms',
+      pages: '../components/04-pages',
     });
   });
 });

--- a/.storybook/setupTwig.test.js
+++ b/.storybook/setupTwig.test.js
@@ -23,8 +23,8 @@ describe('setupTwig', () => {
 
   it('exports emulsifys namespaces', () => {
     expect(namespaces).toEqual({
-      atoms: '../components/01-atoms',
       base: '../components/00-base',
+      atoms: '../components/01-atoms',
       molecules: '../components/02-molecules',
       organisms: '../components/03-organisms',
       templates: '../components/04-templates',

--- a/.storybook/setupTwig.test.js
+++ b/.storybook/setupTwig.test.js
@@ -27,7 +27,7 @@ describe('setupTwig', () => {
       base: '../components/00-base',
       molecules: '../components/02-molecules',
       organisms: '../components/03-organisms',
-      pages: '../components/04-pages',
+      templates: '../components/04-templates',
     });
   });
 });

--- a/package.json
+++ b/package.json
@@ -39,6 +39,7 @@
     "add-attributes-twig-extension": "^0.1.0",
     "bem-twig-extension": "^0.1.1",
     "normalize.css": "^8.0.1",
+    "regenerator-runtime": "^0.13.9",
     "twig-drupal-filters": "^3.1.0"
   },
   "devDependencies": {

--- a/scripts/a11y.test.js
+++ b/scripts/a11y.test.js
@@ -1,3 +1,5 @@
+import 'regenerator-runtime/runtime';
+
 const mockExit = jest
   .spyOn(global.process, 'exit')
   .mockImplementation(() => {});
@@ -17,7 +19,7 @@ const {
   ignore,
   storybookBuildDir,
   pa11y: pa11yConfig,
-} = require('../a11y.config.js');
+} = require('../a11y.config');
 
 const STORYBOOK_BUILD_DIR = path.resolve(__dirname, '../', storybookBuildDir);
 const STORYBOOK_IFRAME = path.join(STORYBOOK_BUILD_DIR, 'iframe.html');

